### PR TITLE
Promote new search engine for kubernetes main github repos

### PR DESCRIPTION
**Pull request by dims**
Etsy/Hound based Search engine maintained by @dims

### CODE REVIEW

**Comment by thockin** [comment]
blocks on #96 ?

**Comment by dims** [comment]
@thockin not really since this just lists the existing instance on GAE. so we can merge this as-is.

**Comment by ixdy** [comment]
looking again at #96, do we want to support `codesearch.` as well as `cs.`?

**Comment by dims** [comment]
@ixdy i'll rework #96 to use `cs` later. let's go with `cs.` (and not add `codesearch.`)

**Comment by dims** [comment]
@ixdy i've closed #96 and am maintaining the code here - https://github.com/dims/k8s-code.appspot.com

So @thockin @ixdy i'll leave it upto you all on whether this PR should be merged as-is.

Thanks,
Dims

File: k8s.io/configmap-nginx.yaml [hunk 1]
```diff
@@ -122,6 +122,13 @@ data:
 
         rewrite ^/(.*)?$    https://github.com/kubernetes/kubernetes/tree/master/$1 redirect;
       }
+      server {
+        server_name cs.kubernetes.io cs.k8s.io;
+        listen 80;
+        listen 443 ssl;
+
+        rewrite ^/$       https://k8s-code.appspot.com redirect;
```
Comment (ixdy) [inline k8s.io/configmap-nginx.yaml]: would it be worth passing along any the rest of the request URI as an argument to the search?

e.g. translate https://cs.k8s.io/foo -> https://k8s-code.appspot.com/?q=foo ?

Comment (dims) [reply k8s.io/configmap-nginx.yaml]: Done! thanks @ixdy

**Comment by ixdy** [comment]
/lgtm

canary tests pass

**Comment by ixdy** [comment]
[https://cs.k8s.io/it%20works!](https://cs.k8s.io/it%20works!)

**Comment by dims** [comment]
Thanks a ton @ixdy !

**Pull request merged**